main/src/test/java/org/mobilitydata/gtfsvalidator/report/model/NoticeViewTest.java

```diff
@@ -0,0 +1,42 @@
+package org.mobilitydata.gtfsvalidator.report.model;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import org.junit.Test;
+import org.mobilitydata.gtfsvalidator.notice.MissingRequiredFieldNotice;
+import org.mobilitydata.gtfsvalidator.notice.Notice;
+
+public class NoticeViewTest {
+  private static final Notice notice = new MissingRequiredFieldNotice("test.txt", 1, "field");
+  private static final NoticeView noticeView = new NoticeView(notice);
+
+  @Test
+  public void getNameTest() {
+    assertEquals(noticeView.getName(), notice.getClass().getSimpleName());
```

Comment:
I think both here and below, I'd avoid just re-using the internal impl code as the expected test value.  Aka, instead of testing against "notice.getClass().getSimpleName()", I'd just hard-code "MissingRequiredFieldNotice" instead.  The general idea being that if getClass().getSimpleName() were to change in some unexpected way (unlikely, I know), then the test would keep passing, but we might start spitting our weird values in our validation report.

Reply:
It makes sense, I will change it.